cdap-examples/WikipediaPipeline/src/main/java/co/cask/cdap/examples/wikipedia/WikipediaPipelineApp.java

```diff
@@ -41,6 +41,7 @@ public void configure() {
     addStream(new Stream(RAW_WIKIPEDIA_STREAM));
     addMapReduce(new StreamToDataset(LIKES_TO_DATASET_MR_NAME));
     addMapReduce(new StreamToDataset(WIKIPEDIA_TO_DATASET_MR_NAME));
+    addMapReduce(new WikipediaDataDownloader());
```

Comment:
As this is an included file, you'll need to update the hash for this file examples/build.sh..after checking this change. Ask Henry how to do this. Tell him jj sent you. :)